gulpfile.ts

```diff
@@ -104,7 +107,6 @@ function buildjQueryUi() {
     "./node_modules/jquery-ui/ui/position.js",
     "./node_modules/jquery-ui/ui/keycode.js",
     "./node_modules/jquery-ui/ui/unique-id.js",
-    "./node_modules/jquery-ui/ui/safe-active-element.js",
```

Comment:
NB: `safe-active-element.js` was removed from `jquery-ui` in https://github.com/jquery/jquery-ui/commit/bb49bd794bc8ea4238162725b518fb46234f3cf9 (released in v1.14.0).